ggshield/config.py

```diff
@@ -170,6 +174,70 @@ def add_ignored_match(self, secret: Dict) -> None:
             self.matches_ignore.append(secret)
 
 
+@dataclass
+class AccountConfig:
+    id: int
+    url: str
+    token: str
+    type: str
+    token_name: str
+    raw_expire_at: InitVar[str]
+    expire_at: Optional[datetime] = None
+
+    def __post_init__(self, raw_expire_at: str) -> None:
+        self.expire_at = datetime.fromisoformat(raw_expire_at)
```

Comment:
Does this mean the key in the file must be named `raw_expire_at`? If so I think we should reverse and name the file key `expire_at` and the parsed key something like `expire_at_datetime`: this avoid propagating current limitations of our implementations to file formats. If in the future we have a way to parse datetime fields without the need of an intermediate one then we can make the switching without having to care about converting config files.

Reply:
I was able to workaround that using only the `expire_at` field.